test/system/CMakeLists.txt

```
@@ -33,4 +33,16 @@ cvc4_add_system_test(sep_log_api)
 cvc4_add_system_test(smt2_compliance)
 cvc4_add_system_test(statistics)
 cvc4_add_system_test(two_smt_engines)
+
+find_package(PythonInterp)
+if(PYTHONINTERP_FOUND)
+  # Add our interactive shell test _only_ if we have Python
+  add_test(
+    NAME interactive_shell
+    COMMAND
+    "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/test/system/interactive_shell.py"
+    WORKING_DIRECTORY "${CMAKE_BINARY_DIR}"
+  )
+endif()
```

`${PYTHON_EXECUTABLE}` should already be [defined](https://github.com/CVC4/CVC4/blob/master/CMakeLists.txt#L348-L354) (and we require Python to build CVC4 anyway), so I think that it would be sufficient to just do this:

```suggestion
add_test(
  NAME interactive_shell
  COMMAND
  "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/test/system/interactive_shell.py"
  WORKING_DIRECTORY "${CMAKE_BINARY_DIR}"
)
```

---

I did it that way because of this: https://github.com/CVC4/CVC4/blob/master/cmake/FindCython.cmake#L38

---

Done!